sway/commands/bar.c

```diff
@@ -82,26 +82,30 @@ struct cmd_results *cmd_bar(int argc, char **argv) {
 		++argv; --argc;
 	}
 
-	if (!config->current_bar && config->reading) {
-		// Create new bar with default values
-		struct bar_config *bar = default_bar_config();
-		if (!bar) {
-			return cmd_results_new(CMD_FAILURE,
-					"Unable to allocate bar state");
-		}
+	if (!config->current_bar) {
+		if (config->reading) {
+			// Create new bar with default values
+			struct bar_config *bar = default_bar_config();
+			if (!bar) {
+				return cmd_results_new(CMD_FAILURE,
+						"Unable to allocate bar state");
+			}
+
+			// set bar id
+			const int len = snprintf(NULL, 0, "bar-%d", config->bars->length - 1) + 1;
```

Comment:
This is over the 80 column limit. It doesn't look like the `const` is actually needed here so that can just get dropped. It'll still be over by 1 character, but to maintain readability, that's close enough.